src/nvim/ex_cmds.lua

```diff
@@ -1814,6 +1814,12 @@ module.cmds = {
     addr_type='ADDR_NONE',
     func='ex_nohlsearch',
   },
+  {
+    command='fclose',
+    flags=bit.bor(BANG, RANGE),
+    addr_type='ADDR_OTHER',
+    func='ex_floatclose',
+  },
```

Comment:
This is a strange place to add this command. It should be next to another command that start with `f`.

Reply:
thanks. Commands are sorted alphabetically. I did not notice T.T